app/src/main/res/layout/fragment_settings.xml

```diff
@@ -1,14 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:layout_margin="24dp"
+    android:orientation="vertical">
 
-    <TextView
+    <LinearLayout
+        android:id="@+id/export_stopwatches_button"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:gravity="center"
-        android:text="settings"
-        android:textColor="#000"
-        android:textSize="22sp"
-        android:textStyle="bold|italic" />
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Export stopwatches data"
```

Comment:
Should be just "Export data to CSV" . Default functionality: export Stopwatches, Categories, Timestamps to three files, deflault name could be something like "stopwatches_<current-unix-timestamp>.csv". If any of the tables is empty it is just not exported at all.